### PR TITLE
feat: support function calling for larksuite send group message

### PR DESCRIPTION
**Pull request by PolarishT**
### Describe what this PR does / why we need it
1. support spring ai alibaba use function calling to invoke api for sending group message

### Does this pull request fix one issue?
#98

### CODE REVIEW

File: community/plugins/spring-ai-alibaba-starter-plugin-larksuite/src/main/java/com/alibaba/cloud/ai/plugin/larksuite/LarkSuiteAutoConfiguration.java [hunk 1]
```diff
@@ -26,16 +26,23 @@
...
-	@ConditionalOnMissingBean
-	@Description("it calls the document api to invoke a method to create a larksuite document")
-	@ConditionalOnProperty(prefix = "spring.ai.alibaba.plugin.larksuite", name = "enabled", havingValue = "true")
-	public LarkSuiteService larksuiteCreateDocFunction(LarkSuiteProperties properties) {
-		return new LarkSuiteService(properties);
-	}
+    @Bean
+    @ConditionalOnMissingBean
+    @Description("it calls the document api to invoke a method to create a larksuite document")
+    @ConditionalOnProperty(prefix = "spring.ai.alibaba.plugin.larksuite", name = "enabled", havingValue = "true")
```
Comment (chickenlj) [inline community/plugins/spring-ai-alibaba-starter-plugin-larksuite/src/main/java/com/alibaba/cloud/ai/plugin/larksuite/LarkSuiteAutoConfiguration.java:36]: We can put this annotation on LarkSuiteAutoConfiguration class.

File: community/plugins/spring-ai-alibaba-starter-plugin-larksuite/src/main/java/com/alibaba/cloud/ai/plugin/larksuite/LarkSuiteAutoConfiguration.java [hunk 2]
```diff
@@ -26,16 +26,23 @@
...
+    public LarkSuiteDocService larksuiteCreateDocFunction(LarkSuiteProperties properties) {
+        return new LarkSuiteDocService(properties);
+    }
 
+    @Bean
+    @ConditionalOnMissingBean
+    @Description("it runs a api to invoke a method to send group message")
```
Comment (chickenlj) [inline community/plugins/spring-ai-alibaba-starter-plugin-larksuite/src/main/java/com/alibaba/cloud/ai/plugin/larksuite/LarkSuiteAutoConfiguration.java:43]: Do we need to add `larksuite` keyword in the description for LLM to better understand that this is a Larksuite messaging tool?

**Review by chickenlj — APPROVED** [review]
LGTM.

**Comment by chickenlj** [comment]
Please run `mvn javaformat:apply` to format the code.

**Comment by PolarishT** [comment]
> Please run `mvn javaformat:apply` to format the code.

done

File: community/plugins/spring-ai-alibaba-starter-plugin-larksuite/src/main/java/com/alibaba/cloud/ai/plugin/larksuite/LarkSuiteAutoConfiguration.java (hunk 2 above)
Comment (PolarishT) [reply community/plugins/spring-ai-alibaba-starter-plugin-larksuite/src/main/java/com/alibaba/cloud/ai/plugin/larksuite/LarkSuiteAutoConfiguration.java]: yes ,of course ,each proxy bean have different obligatory

**Pull request closed without merging**